stories/dropdown.stories.ts

```diff
@@ -156,269 +160,251 @@ interface IgcDropdownArgs {
-          .sameWidth=${sameWidth}
-          .positionStrategy=${positionStrategy}
-        >
-          <igc-dropdown-group>
-            <h3 slot="label">Research & Development</h3>
-            ${items
-              .slice(0, 3)
-              .map(
-                (item) => html`<igc-dropdown-item>${item}</igc-dropdown-item>`
-              )}
-          </igc-dropdown-group>
-          <igc-dropdown-group>
-            <h3 slot="label">Product Guidance</h3>
-            ${items
-              .slice(3, 5)
-              .map(
-                (item) => html`<igc-dropdown-item>${item}</igc-dropdown-item>`
-              )}
-          </igc-dropdown-group>
-          <igc-dropdown-group>
-            <h3 slot="label">Release Engineering</h3>
-            <igc-dropdown-item
-              ><igc-icon slot="prefix" name="home"></igc-icon>${items[5]}<span
-                slot="suffix"
-                >-</span
-              ></igc-dropdown-item
-            >
-          </igc-dropdown-group>
-        </igc-dropdown>
-      </div>
-    </div>
+      <igc-dropdown-group>
+        <p class="group-title" slot="label">
+          UN Region: <strong>Europe</strong>
+        </p>
+        <igc-dropdown-header>Estimate for 2023</igc-dropdown-header>
+        ${gdpEurope}
+      </igc-dropdown-group>
 
-    <igc-dropdown
-      id="ddl3"
-      style="align-self: center;"
-      distance=${distance}
-      .open=${open}
-      .flip=${flip}
-      .keepOpenOnOutsideClick=${keepOpenOnOutsideClick}
-      .placement=${placement}
-      .scrollStrategy=${scrollStrategy}
-      .sameWidth=${sameWidth}
-      .positionStrategy=${positionStrategy}
-    >
-      <igc-button slot="target">Dropdown 3</igc-button>
-      ${items.map(
-        (item) => html`<igc-dropdown-item>${item}</igc-dropdown-item>`
-      )}
+      <igc-dropdown-group>
+        <p slot="label" class="group-title">
+          UN Region: <strong>Americas</strong>
+        </p>
+        <igc-dropdown-header>Estimate for 2023</igc-dropdown-header>
+        ${gdpAmericas}
+      </igc-dropdown-group>
     </igc-dropdown>
+  `,
+};
+
+export const WithNonSlottedTarget: Story = {
+  render: ({
+    distance,
+    open,
+    flip,
+    keepOpenOnOutsideClick,
+    keepOpenOnSelect,
+    placement,
+    positionStrategy,
+    sameWidth,
+  }) => html`
+    <style>
+      .container {
+        display: flex;
+        justify-content: space-between;
+      }
+    </style>
+    <div class="container">
+      <igc-button id="1st" onclick="dropdown.show('1st')">First</igc-button>
+      <igc-button id="2nd" onclick="dropdown.show('2nd')">Second</igc-button>
+      <igc-button id="3rd" onclick="dropdown.show('3rd')">Third</igc-button>
+      <igc-button id="4th" onclick="dropdown.show('4th')">Fourth</igc-button>
+    </div>
+    <igc-input
+      id="input"
+      style="max-width: 15rem"
+      label="Focus me"
+      onfocus="dropdown.show('input')"
+    ></igc-input>
```

Comment:
There seems to be an issue with this setup of the buttons and input. None of the dropdowns would open on click. The following error is logged: "dropdown.show is not a function
    at IgcButtonComponent.onclick".

Reply:
I actually still reproduce this on the 'Docs' page but no on the story page.

Reply:
Should be fixed now.